classes/Configuration.php

```diff
@@ -131,13 +143,21 @@ public static function configurationIsLoaded()
         return self::$_initialized;
     }
 
+    /**
+     * @deprecated 8.0.0 Use resetStaticCache method instead.
+     */
+    public static function clearConfigurationCacheForTesting()
```

Comment:
Could you deprecate method like in doc with `trigger_error` ?
https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations